content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,126 @@
+
+What's next? We're developing a new built-in mechanism to help limit Pod privileges, with a working title of “PSP Replacement Policy.” Our plan is for this new mechanism to cover the key PodSecurityPolicy use cases, with greatly improved ergonomics and maintainability. To learn more, read [PodSecurityPolicy Deprecation: Past, Present, and Future](/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future).
+
+### TopologyKeys Deprecation
+The Service field `topologyKeys` is now deprecated; all the component features that used this field were previously alpha, and are now also deprecated.
+We've replaced `topologyKeys` with a way to implement topology-aware routing, called topology-aware hints.  Topology-aware hints are an alpha feature in Kubernetes 1.21. You can read more details about the replacement feature in [Topology Aware Hints](/docs/concepts/services-networking/service-topology/); the related [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/2433-topology-aware-hints/README.md) explains the context for why we switched.
+
+## Other Updates
+
+### Graduated to Stable
+
+* [EndpointSlice](https://github.com/kubernetes/enhancements/issues/752)
+* [Add sysctl support](https://github.com/kubernetes/enhancements/issues/34)
+* [PodDisruptionBudgets](https://github.com/kubernetes/enhancements/issues/85)
+
+### Notable Feature Updates
+
+* [External client-go credential providers](https://github.com/kubernetes/enhancements/issues/541) - beta in 1.21
+* [Structured logging](https://github.com/kubernetes/enhancements/issues/1602) - graduating to beta in 1.22
+* [TTL after finish cleanup for Jobs and Pods](https://github.com/kubernetes/enhancements/issues/592) - graduated to beta
+
+# Release notes
+
+You can check out the full details of the 1.21 release in the [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md).
+
+# Availability of release
+
+Kubernetes 1.21 is available for [download on GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.21.0). There are some great resources out there for getting started with Kubernetes. You can check out some [interactive tutorials](https://kubernetes.io/docs/tutorials/) on the main Kubernetes site, or run a local cluster on your machine using Docker containers with [kind](https://kind.sigs.k8s.io). If you’d like to try building a cluster from scratch, check out the [Kubernetes the Hard Way](https://github.com/kelseyhightower/kubernetes-the-hard-way) tutorial by Kelsey Hightower.
+
+# Release Team
+
+This release was made possible by a very dedicated group of individuals, who came together as a team in the midst of a lot of things happening out in the world. A huge thank you to the release lead Nabarun Pal, and to everyone else on the release team for supporting each other, and working so hard to deliver the 1.21 release for the community.
+
+# Release Logo
+
+![Kubernetes 1.21 Release Logo](XXXX)
+
+
```

Comment:
Delete extra whitespace
```suggestion
```

Reply:
Just pushed an update to @divya-mohan0209 's branch to address the extra whitespace :)